src/mobu/models/business/__init__.py

```diff
@@ -0,0 +1,17 @@
+from typing import TypeAlias
+
+from ...models.business.empty import EmptyLoopConfig
+from ...models.business.gitlfs import GitLFSConfig
+from ...models.business.notebookrunner import NotebookRunnerConfig
+from ...models.business.nubladopythonloop import NubladoPythonLoopConfig
+from ...models.business.tapqueryrunner import TAPQueryRunnerConfig
+from ...models.business.tapquerysetrunner import TAPQuerySetRunnerConfig
+
+BusinessConfig: TypeAlias = (
+    TAPQueryRunnerConfig
+    | EmptyLoopConfig
+    | GitLFSConfig
+    | NotebookRunnerConfig
+    | NubladoPythonLoopConfig
+    | TAPQuerySetRunnerConfig
+)
```

Comment:
Last time I was working with the list of Config union types, Russ gave me some good reason EmptyLoopConfig should come last.  So I think it should.

Reply:
Done!  But:
* I'm very curious about the reason.  The only thing I can think of is that it has to do with the way that Pydantic tries to [validate unions](https://docs.pydantic.dev/latest/concepts/unions/), and maybe we were depending on V1 defaults of left-to-right validation, which would somehow end with trying EmptyLoop?
* I just changed the validation to use the `discriminator` functionality based on the `type` field, so now maybe it doesn't matter?

Reply:
Yep, looks like [that's the fear](https://github.com/lsst-sqre/mobu/pull/339#discussion_r1526696004).
* With the `discriminator` functionality, that doesn't apply anymore
* Even before that, with PydanticV2, the default changed to `smart` from `left_to_right`, so I _think_ it would pick the most explicit match from among the members